features/kernel/src/main/feature/feature.xml

```diff
@@ -64,14 +64,15 @@
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
     <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
-        <feature version="[6.0,6.1)">pax-http</feature>
+        <feature version="[7.2,7.3)">pax-http</feature>
         <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.5.1</bundle>
         <bundle start-level="30" dependency="true">mvn:javax.el/javax.el-api/3.0.0</bundle>
-        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${pax.web.version}</bundle>-->
+        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${project.version}</bundle>-->
```

Comment:
❓ Was the `pax.web.version` -> `project.version` change intentional here?

Reply:
❓ Why is the override for the `pax-http-whiteboard` feature now removed?

Reply:
@clockard pointed out that the `7.2.3` version of this feature does not have the vulnerability, so we don't need to override it anymore. https://github.com/codice/ddf/pull/3635#discussion_r212395785